swagger.json

```diff
@@ -4362,6 +4362,37 @@
                 }
             ]
         },
+        "TaggedAddressBase": {
+            "type": "object",
+            "required": [
+                "id"
+            ],
+            "properties": {
+                "id": {
+                    "type": "integer",
+                    "format": "int64",
+                    "description": "The ID of the address being tagged.",
+                    "example": 345
+                }
+            }
+        },
+        "TaggedAddress": {
+            "allOf": [
+                {
+                    "$ref": "#/definitions/TaggedAddressBase"
+                },
+                {
+                    "type": "object",
+                    "properties": {
+                        "name": {
```

Comment:
Should `name` be required?

Reply:
we don't require `name` in the other `Tagged<object>` definitions.

Reply:
Ok, we can fix them all in a separate PR.